src/components/Header/index.js

```diff
@@ -0,0 +1,3 @@
+import HeaderLeft from './HeaderLeft';
```

Comment:
Component name should be same as the filename for a component file. So Please name this "Header".